setup.cfg

```diff
@@ -0,0 +1,6 @@
+[versioneer]
+VCS = git
+style = pep440
+versionfile_source = pygeos/_version.py
+versionfile_build = pygeos/_version.py
+tag_prefix =
```

Comment:
Compared to what we have in pandas/geopandas, there we also have a `parentdir_prefix` entry. 
But, I don't really understand what this is supposed to do (and tested this branch, which seems to work correctly to set the version and to create sdists)

Reply:
I tested with and without and saw no difference either.. also not sure what it is about.